src/redux/books/books.js

```diff
@@ -1,38 +1,31 @@
-import { v4 as uuidv4 } from 'uuid';
+import { createAsyncThunk } from '@reduxjs/toolkit';
 
 // Actions
 const ADD = 'bookstore-react/booksReducer/ADD';
 const REMOVE = 'bookstore-react/booksReducer/REMOVE';
+const GET = 'bookstore-react/booksReducer/GET';
 
-// Array with a few books
-const listBooks = [
-  {
-    id: uuidv4(),
-    title: 'The Hunger Games',
-    author: 'Suzanne Collins',
-  },
-  {
-    id: uuidv4(),
-    title: 'Dune',
-    author: 'Frank Herbert',
-  },
-  {
-    id: uuidv4(),
-    title: 'Capital in the Twenty-First Century',
-    author: 'Suzanne Collins',
-  },
-];
+// API URL
+const url = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/xKgONp1vYVZJzmgq5zMK/books';
 
 // Reducer
-export default function booksReducer(state = listBooks, action) {
+export default function booksReducer(state = [], action) {
   switch (action.type) {
-    case ADD:
-      return [
-        ...state,
-        action.book,
-      ];
-    case REMOVE:
-      return state.filter((book) => book.id !== action.book.id);
+    case `${ADD}/fulfilled`:
+      return state.concat(action.meta.arg);
+    case `${REMOVE}/fulfilled`:
+      return state.filter((book) => book.item_id !== action.meta.arg);
+    case `${GET}/fulfilled`:
+    // console.log('chewie', action.payload);
```

Comment:
- [ ] @paulinagonzalezc great work on your reducer function but you forgot to remove the commented line, it's not good practice to have it in our codebase.

Reply:
Thank you @danmainah, I will remove it.